README
=STANDORTE=

Eingabe der Standorte im Backend

==RealURL==
'fakultaet' => array(
					array(
						'GETvar' => 'tx_standorte_pi1[fakultaetUid]',
						'lookUpTable' => array(
							'table' => 'tx_standorte_domain_model_fakultaet',
							'id_field' => 'uid',
							'alias_field' => 'titel',
							'maxLength' => 50,
							'addWhereClause' => 'AND NOT deleted',
							'useUniqueCache' => 1,
							'useUniqueCache_conf' => array(
								'strtolower' => 1,
								'spaceCharacter' => '_',
							),
						),
					),
					array(
						'GETvar' => 'tx_standorte_pi1[action]'
					),
					array(
						'GETvar' => 'tx_standorte_pi1[controller]'
					),
				),

==TypoScript==

* Storage Folder anlegen
* In Constants des TS:
	tx_schulungen.persistence.storagePid = 
* Im Setup das mitgelieferte TypoScript einbinden

ChangeLog
2012-07-17 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Added pizNr for buildings

2012-03-12 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Titel fuer die Sortierung entfernt und Sortierung nach Sigeln vorgenommen (WWW-1144)
	* Layout der Single Ansicht optimiert (WWW-1102)

2012-03-05 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Titel der Detailseiten geaendert (WWW-1103)
	* Linkbezeichnung geaendert (WWW-1102)
	* Manual aktualisiert

2012-02-01 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Layout der Einzelansicht verbessert (WWW-1102)

2011-12-20 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Sigelliste angepasst (WWW-1047)
	* Aktuelle, kompatible Library integriert (WWW-1047)

2011-11-28 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Versionierung hinzugefuegt

2011-11-24 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Verlinkung zu den Bibliothekswebsites angepasst (WWW-909)

2011-11-23 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Suche im Backend TYPO3 4.6. kompatibel gestaltet (WWW-896)

2011-11-09 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Einfuegen der Strasse in die Liste der Links fuer Antje Niemann (WWW-657)

2011-10-31 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Anpassung der Links auf die Bibliotheken im vorgegebenen Schnema (WWW-657)

2011-10-12 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* CSS Formatierung bei der Listenansicht optimiert

2011-09-28 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Cleanups und Erweiterungen

2011-09-22 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Anpassung der Bibliotheksliste an die Vorgaben aus dem CSV (WWW-657)
	* Optimierungen im Code

2011-09-21 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Neue Action listBibMitLink zur einfachen Ausgabe der Links und Titel der Bibliotheken (WWW-657)

2011-09-02 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Optimierung der CSS Einbindung
	* Optimierung der Js Einbindung
	* CSS Anpassung bei der Sigel/Bibliothekslistenansicht
	* Ausgabe der Sigel stets mit einem 7/ vor dem Sigel aus der Datenbank

2011-08-31 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Anpassung der Datenbanken

2011-05-09 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Anpassung ViewHelper an Title Tag
	* Aenderung Ausgaben bezueglich der Linktitel(WWW-667)

2011-04-01 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* RTE Einstellungen gefixt (WWW-621)

2011-03-23 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* UpperCamelCase bei Templates
	* Einbindung aktuellerer jQuery Bibliothek im Backend
	* Optimierung Nachfrage beim Loeschen der Bibliothek

2011-03-21 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Deprecated Funktion entfernt im Hook

2011-03-17 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Workaround fuer TYPO3 4.4 (Repository)

2011-03-16 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Layout der Bibliotheken optimiert
	* CSS und JS angepasst
	* Korrektur der Sortierreihenfolge (WWW-605)
	* Bugfix in Singleansicht (mehr als 100 Funktionen aufgerufen)
	* Bei ehemaligen Bibliotheken ohne Adresse wird das Adressfeld ausgeblendet (WWW-612)

2011-03-15 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Der Bestand ist teilweise im GUK enthalten (WWW-596)

2011-03-11 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Oeffnungszeiten als Freitext - keine Berechnung mehr ob geoeffnet oder geschlossen (WWW-607)
	* Aenderung der Ausgabereihenfolge im FE (Sigel, Anschrift, Oeffnungszeiten, Kontakt, Bestand, Zusatzinformationen) (WWW-604)
	* Optimierung Zuweisung Controller->View
	* Lokalisierung (WWW-606)

2011-03-08 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Einfuegen der Storage PID ueber Constants

2011-02-28 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Inhaltsverzeichnis in der Sidebar wird nicht angezeigt (WWW-581)
	* Aenderung der Zeitzone

2011-02-23 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Hook Bugfixing

2011-02-22 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Hook Anpassung

2011-02-11 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Fakultaeten und Bibliotheken werden mit Nachfrage geloescht

2011-02-10 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Pi2 erstellt und konfiguriert (Bibliotheken und Sigelauflistung) WWW-525
	* Singleansicht in Pi2 integriert (Klick auf Bibliothek) WWW-531
	* Loeschen von Bibliotheken moeglich (Flag deleted=1 in DB) WWW-517

2011-02-09 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Codeoptimierung
	* Pruefung ob in Semesterferien andere Oeffnungszeiten (WWW-527)

2011-02-04 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Hookaufruf kompatibel zu TYPO3 4.4 gemacht
	* Content Rendering vereinheitlicht (WWW-530)
	* Maps Optik angepasst (Infobox)
	* Singleansicht (WWW-531)
	* Ansicht nur Bibliotheken mit Sigel und Titel (WWW-525)

2011-02-03 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Frontendausgabe angepasst (WWW-529)
	* Ausgabe externer Links im neuen Fenster
	* Optimiertes Parsen der Links
	* Standardmarker fuer Google Maps Einbindung (WWW-524)
	* Onclick Oeffnung des Maps Infofensters
	* Sigel generischer gestaltet (WWW-526)
	* Zeitzonenhartkodierung entfernt

2011-01-28 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Geocoding optimierung
	* Verbesserung der Performance
	* Maps funktioniert auch fuer Bibliotheken ausserhalb Deutschlands (Beispiel China)
	* Aus Kompatibilitaetsgruenden Hook deaktiviert

2011-01-26 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* jQuery Improvements
	* Update Controller
	* Update Ajax Funktionen

2011-01-26 Ingo Pfennigstorf <pfennigstorf@sub.uni-goettingen.de>
	* Optimierung Backend Modul
	* Tabellensortierung


ext_emconf.php
<?php

########################################################################
# Extension Manager/Repository config file for ext "standorte".
#
# Auto generated 13-03-2012 08:27
#
# Manual updates:
# Only the data in the array - everything else is removed by next
# writing. "version" and "dependencies" must not be touched!
########################################################################

$EM_CONF[$_EXTKEY] = array(
	'title' => 'Standorte der SUB Goettingen',
	'description' => 'Anzeige der Standorte der SUB Goettingen inkl. Google Maps usw',
	'category' => 'plugin',
	'shy' => 0,
	'version' => '2.0.0',
	'dependencies' => 'extbase,fluid',
	'conflicts' => '',
	'priority' => '',
	'loadOrder' => '',
	'module' => '',
	'state' => 'beta',
	'uploadfolder' => 1,
	'createDirs' => '',
	'modify_tables' => '',
	'clearcacheonload' => 1,
	'lockType' => '',
	'author' => 'Ingo Pfennigstorf',
	'author_email' => 'pfennigstorf@sub.uni-goettingen.de',
	'author_company' => 'Goettingen State and University Library, Germany http://www.sub.uni-goettingen.de',
	'CGLcompliance' => '',
	'CGLcompliance_note' => '',
	'constraints' => array(
		'depends' => array(
			'extbase' => '0.0.0-0.0.0',
			'fluid' => '0.0.0-0.0.0',
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
	'_md5_values_when_last_written' => 'a:64:{s:9:"ChangeLog";s:4:"9d7e";s:6:"README";s:4:"adc1";s:9:"build.xml";s:4:"6661";s:12:"ext_icon.gif";s:4:"64af";s:17:"ext_localconf.php";s:4:"5909";s:14:"ext_tables.php";s:4:"c2a1";s:14:"ext_tables.sql";s:4:"e0d4";s:16:"locallang_db.xml";s:4:"7ca4";s:12:"t3jquery.txt";s:4:"3353";s:40:"Classes/Controller/BackendController.php";s:4:"ae1c";s:43:"Classes/Controller/BibliothekController.php";s:4:"ee22";s:42:"Classes/Controller/FakultaetController.php";s:4:"1b76";s:35:"Classes/Domain/Model/Bibliothek.php";s:4:"3f45";s:34:"Classes/Domain/Model/Fakultaet.php";s:4:"225e";s:50:"Classes/Domain/Repository/BibliothekRepository.php";s:4:"2a8b";s:49:"Classes/Domain/Repository/FakultaetRepository.php";s:4:"a5c5";s:25:"Classes/Hooks/Sidebar.php";s:4:"182e";s:39:"Classes/ViewHelpers/SigelViewHelper.php";s:4:"cca8";s:42:"Classes/ViewHelpers/TypolinkViewHelper.php";s:4:"4675";s:58:"Configuration/TCA/tx_standorte_domain_model_bibliothek.php";s:4:"95a2";s:57:"Configuration/TCA/tx_standorte_domain_model_fakultaet.php";s:4:"afc2";s:34:"Configuration/TypoScript/setup.txt";s:4:"6f2a";s:31:"Documentation/Manual/Manual.rst";s:4:"a0b6";s:41:"Resources/Private/Language/_locallang.xml";s:4:"d278";s:43:"Resources/Private/Language/de.locallang.xlf";s:4:"9105";s:43:"Resources/Private/Language/en.locallang.xlf";s:4:"e5ac";s:44:"Resources/Private/Language/locallang.mod.xml";s:4:"37b7";s:44:"Resources/Private/Layouts/backendLayout.html";s:4:"47af";s:44:"Resources/Private/Layouts/defaultLayout.html";s:4:"8ff0";s:42:"Resources/Private/Partials/Bibliothek.html";s:4:"ac9a";s:41:"Resources/Private/Partials/backendJs.html";s:4:"b9a4";s:39:"Resources/Private/Partials/bestand.html";s:4:"7108";s:36:"Resources/Private/Partials/bild.html";s:4:"4311";s:37:"Resources/Private/Partials/sigel.html";s:4:"768f";s:37:"Resources/Private/Sass/Standorte.scss";s:4:"a416";s:57:"Resources/Private/Templates/Backend/DeleteBibliothek.html";s:4:"3b0c";s:46:"Resources/Private/Templates/Backend/Index.html";s:4:"b22f";s:57:"Resources/Private/Templates/Backend/ListBibliotheken.html";s:4:"4c88";s:68:"Resources/Private/Templates/Backend/ListBibliothekenByFakultaet.html";s:4:"9975";s:56:"Resources/Private/Templates/Backend/ListFakultaeten.html";s:4:"add8";s:48:"Resources/Private/Templates/Bibliothek/List.html";s:4:"6f92";s:58:"Resources/Private/Templates/Bibliothek/ListBibMitLink.html";s:4:"982c";s:58:"Resources/Private/Templates/Bibliothek/ListSigelTitel.html";s:4:"74cc";s:50:"Resources/Private/Templates/Bibliothek/Single.html";s:4:"24fd";s:48:"Resources/Private/Templates/Fakultaet/Index.html";s:4:"66f4";s:47:"Resources/Private/Templates/Fakultaet/List.html";s:4:"ffe9";s:34:"Resources/Public/Css/Standorte.css";s:4:"8f1c";s:32:"Resources/Public/Css/backend.css";s:4:"13c6";s:45:"Resources/Public/Js/jquery.tablesorter.min.js";s:4:"94e7";s:43:"Resources/Public/Js/jquery.uitablefilter.js";s:4:"97f9";s:27:"Resources/Public/Js/maps.js";s:4:"b509";s:29:"Resources/Public/eId/maps.php";s:4:"9933";s:28:"Resources/Public/img/asc.gif";s:4:"f8a1";s:27:"Resources/Public/img/bg.gif";s:4:"c01a";s:31:"Resources/Public/img/closed.gif";s:4:"acbf";s:29:"Resources/Public/img/desc.gif";s:4:"a548";s:66:"Resources/Public/img/icon_tx_standorte_domain_model_bibliothek.png";s:4:"0ba8";s:65:"Resources/Public/img/icon_tx_standorte_domain_model_fakultaet.png";s:4:"99be";s:71:"Resources/Public/img/icon_tx_standorte_domain_model_oeffnungszeiten.png";s:4:"f1ca";s:31:"Resources/Public/img/opened.gif";s:4:"d7ba";s:34:"Resources/Public/img/standorte.png";s:4:"3ea3";s:50:"Tests/Unit/Controller/BibliothekControllerTest.php";s:4:"ec97";s:15:"build/phpcs.xml";s:4:"ab01";s:15:"build/phpmd.xml";s:4:"ab48";}',
	'suggests' => array(
	),
);

?>